reference/conanfile/attributes.rst

```diff
@@ -933,30 +933,58 @@ Components
 
     This is a **experimental** feature subject to breaking changes in future releases.
 
```

Comment:
IMO, I'd remove all this "Components" section and substitute it by a paragraph (if you feel like, move more things to the other section):

_"More fine grained control over individual libraries available in a single Conan package can be achieved using `components` (LINK). Components allow you define a `cpp_info` object per each of those libraries and also requirements between them and to components of other packages:"_

>>>> Python with example. 

_"Note that components should declare explicitly that they require other packages...."_

Reply:
Kind of agree. This section still reads a bit like tutorial, not reference. Maybe moving some parts to the other doc, and leaving here just the components fields reference.